ts/components/SetupWidget.tsx

```diff
@@ -163,7 +163,7 @@ export default class SetupWidget extends React.Component<SetupWidgetProps, Setup
 
 		socket.on('connect', () => {
 			interaction.startGame();
-			this.props.onSetup(sink, interaction);
+			this.props.onSetup(sink, null, null, interaction);
```

Comment:
@beheh websocketed replays cannot be downloaded this way right?

Reply:
Well, technically they could, but we don't use them anywhere. This is fine.